src/App.js

```diff
@@ -0,0 +1,70 @@
+import { render } from "@testing-library/react";
+import "./App.css";
+import Select from "./DropDown";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Title from "./Title";
+import Button from "./Button";
+import WeatherForcastCard from "./WeatherForcastCard";
+
+function App() {
+  const [weatherData, setWeatherData] = useState("");
```

Comment:
```suggestion
  const [weatherData, setWeatherData] = useState();
```

初期値は空文字ではなくなし(undefined)が適当かと思います